kits/debate-arena/.env.example

```
@@ -0,0 +1,9 @@
+# Lamatic API credentials (Settings -> API Keys / Project / API Docs in Studio)
+LAMATIC_API_URL=YOUR_API_ENDPOINT
+LAMATIC_PROJECT_ID=YOUR_PROJECT_ID
+LAMATIC_API_KEY=YOUR_API_KEY
+
+# Deployed flow IDs (Studio -> flow -> three-dot menu -> Copy Flow Id)
+DEBATE_SETUP_FLOW_ID=347d6354-ffe4-48eb-b1f2-e3d08cde75ab
+DEBATE_ROUND_FLOW_ID=7f2718fa-28c4-41c4-a14a-ec3e5fd523ab
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEBATE_ROUND_FLOW_ID key should go before the DEBATE_SETUP_FLOW_ID key

(UnorderedKey)